explorer/jest.setup.ts

```diff
@@ -10,5 +10,7 @@ beforeAll(async () => {
 })
 afterEach(() => clearDb())
 afterAll(async () => {
-  await globalDBConnection.close()
+  if (globalDBConnection) {
```

Comment:
This should always exist no?

Reply:
Maybe we should remove `globalDBConnection` altogether and replace it with `getConnection().close()`. It'll throw an error that way too that its not connected

Reply:
I don't think `getConnection().close()` actually closes the pool, just that connection.

Reply:
The connection is the pool 
![image](https://user-images.githubusercontent.com/6404866/79509148-f2cf7f80-8008-11ea-9a57-089ce0038e38.png)